sharedse/src/commonJvmMain/kotlin/com/ustadmobile/port/sharedse/contentformats/VideoTypePlugin.kt

```diff
@@ -0,0 +1,23 @@
+package com.ustadmobile.port.sharedse.contentformats
+
+import com.ustadmobile.core.catalog.contenttype.VideoType
+import com.ustadmobile.lib.db.entities.ContentEntryWithLanguage
+import com.ustadmobile.lib.db.entities.ContentEntry
+import com.ustadmobile.lib.db.entities.Language
+import com.ustadmobile.port.sharedse.contentformats.ContentTypePlugin
+
+import java.io.File
+
+class VideoTypePlugin : VideoType(), ContentTypePlugin {
+
+    override fun getContentEntry(file: File): ContentEntryWithLanguage? {
+        return ContentEntryWithLanguage().apply {
```

Comment:
This needs to check that the given file is actually a video file. For now let's do this by simply checking the file extension is a known video extension.